### PR TITLE
Do a path search for grc even if it is an alias

### PR DESCRIPTION
**Pull request by satakeh**

### CODE REVIEW

File: warhol.plugin.zsh [hunk 1]
```diff
@@ -3,145 +3,138 @@
...
   fi
   if [ -x /bin/traceroute6 ]; then
     function traceroute6(){
-      grc --colour=auto /bin/traceroute6 "$@"
+      \grc --colour=auto /bin/traceroute6 "$@"
     }
   fi
 
 fi
-
```
Comment (unixorn) [inline warhol.plugin.zsh]: Removing LSCOLORS and LS_COLORS should be a separate PR so it can be discussed.

**Comment by unixorn** [comment]
Removing the LSCOLORS & LS_COLORS should go in a different PR. They're there to provide a consistent set of colors across both GNU and BSD ls. If you want different ones, you can set your own values after your framework loads your plugins.

**Comment by satakeh** [comment]
Oops. I wasn't intended to include that commit in the PR.

Now I fixed it, sorry.

**Comment by unixorn** [comment]
No problem, merging. Thanks for the contribution.

If you want to open an issue about setting LSCOLORS/LS_COLORS we can discuss removing them there.
